core/types/Cargo.toml

```diff
@@ -18,12 +18,14 @@ alloc = []
 
 [dependencies]
 bitflags = "2.0.0"
+constant_time_derive = { path = "../../constant_time_derive", version = "0.1.0" }
 displaydoc = { version = "0.2.3", default-features = false }
 mc-sgx-core-sys-types = { path = "../sys/types", version = "=0.5.1-beta.0" }
 mc-sgx-util = { path = "../../util", version = "=0.5.1-beta.0" }
 nom = { version = "7.1.2", default-features = false }
 rand_core = { version = "0.6.4", default-features = false }
 serde = { version = "1.0.152", default-features = false, features = ["derive"], optional = true }
+subtle = { version = "2.4.0", default-features = false, features = ["i128"] }
```

Comment:
:question: I'm not sure we want to pull in i128, it's not available on all platforms and we want the types to work on HW wallets.
It may be that we want CT behind feature flags, but it still seems like it might be an attack vector if a malicious app times how long it takes a HW wallet to verify.

Reply:
@jcape said we can change it if we come across a problem. @jcape thoughts?